assets/scss/layouts/header/_header.scss

```diff
@@ -173,16 +176,12 @@
 .header-logo-image-container:after {
     content: '';
     display: block;
-    padding-bottom: remCalc($header-height) - $header-logo-marginVertical * 2;
-
-    @include breakpoint("medium") {
-        padding-bottom: get-height(stencilString('logo_size'));
-    }
 }
 
 .header-logo-image {
-    @include lazy-loaded-img;
```

Comment:
Why do we remove lazy-loaded-img? It set a placeholder while image is loading

Reply:
because it has absolute styles for logo image, which we can not use, because from image size depends link width and its outline. Now he have fixed size for header, so we do not need to reserve space for image anymore

Reply:
On screen with 745px space won't be allocated

Reply:
Do not understand clearly your question, but on 745px width logo looks fine

![Screenshot 2020-08-28 at 15 29 29](https://user-images.githubusercontent.com/66325265/91561050-8fa11280-e943-11ea-8838-2e87784822b3.png)

Reply:
What will happen with image placeholder on this resolution?

Reply:
I suppose everything will be okay, because we have fixed header height. Image in the header. So page content will not be jumping while image loading

Reply:
Ok, thanks for investigation

Reply:
thanks for attentive review